scripts/zones/Southern_San_dOria/npcs/Gondebaud.lua

```diff
@@ -0,0 +1,61 @@
+-----------------------------------
+-- Area: Southern San d'Oria
+-- NPC: Gondebaud
+-- Trust NPC
+-- !pos 123.754 0.000 92.125 230
+-----------------------------------
+local ID = require("scripts/zones/Southern_San_dOria/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
+require("scripts/globals/quests")
+require("scripts/globals/settings")
+require("scripts/globals/trust")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+    local hasPermit = player:hasKeyItem(tpz.ki.WINDURST_TRUST_PERMIT) or
+                      player:hasKeyItem(tpz.ki.BASTOK_TRUST_PERMIT) or
+                      player:hasKeyItem(tpz.ki.SAN_DORIA_TRUST_PERMIT)
+
+    if hasPermit and trade:getSlotCount() == 1 and tpz.trust.cipherToSpellID[trade:getItemId(0)] ~= nil then
+        player:setLocalVar("TradingTrustCipher", tpz.trust.cipherToSpellID[trade:getItemId(0)])
+        local fameModifier = 0
+        player:startEvent(3503, 0, 0, fameModifier, trade:getItemId(0))
+    end
+end
+
+function onTrigger(player,npc)
+    local TrustSandoria = player:getQuestStatus(SANDORIA, tpz.quest.id.sandoria.TRUST_SANDORIA)
+	local TrustBastok   = player:getQuestStatus(BASTOK, tpz.quest.id.bastok.TRUST_BASTOK)
+	local TrustWindurst = player:getQuestStatus(WINDURST, tpz.quest.id.windurst.TRUST_WINDURST)
+    local lvl = player:getMainLvl()
+
+    if lvl >= 5 and TrustSandoria == QUEST_AVAILABLE and (TrustWindurst == QUEST_COMPLETED or TrustBastok == QUEST_COMPLETED) then
+		player:startEvent(3504)
+    elseif ENABLE_TRUST_QUESTS == 1 and lvl >= 5 and TrustSandoria == QUEST_AVAILABLE and TrustWindurst == QUEST_AVAILABLE and TrustBastok == QUEST_AVAILABLE then
+        -- TODO: Heard/Not Heard
+        player:startEvent(3500, 1, 1, 1, 1)
+    elseif player:hasKeyItem(tpz.ki.RED_INSTITUTE_CARD) then
+        player:startEvent(3501)
+    elseif TrustSandoria == QUEST_COMPLETED then
+		player:startEvent(3502)
```

Comment:
To mark as a TODO: This event will point in the player in the direction of other nation-based trusts they can find (that the player qualifies for). It will probably need either params or an event update (I captured this with an ancient version of EventView which didn't display updates, so would need to check the PV logs).